yarn.lock

```diff
@@ -11704,6 +11704,11 @@ fill-range@^7.0.1:
   dependencies:
     to-regex-range "^5.0.1"
 
+filter-obj@^1.1.0:
```

Comment:
😬 why do we have yarn.lock and package-json.lock? Are we using both yarn and npm?

Reply:
I only have yarn.lock
I'm not using npm, are you